corehq/apps/app_manager/static/app_manager/spec/download_async_modal_spec.js

```diff
@@ -0,0 +1,84 @@
+/* global chai, $, sinon */
+
+describe('Async Download Modal', function() {
+    var downloader = null,
+        expect = chai.expect,
+        url = 'test_url';
+
+    describe('#AsyncDownloader.isDone', function() {
+        var modal = $('#modal'),
+            download_poll_id = '12345';
+
+        beforeEach(function() {
+            downloader = new AsyncDownloader(modal, url);
+            downloader.download_poll_id = download_poll_id;
+        });
+
+        afterEach(function() {
+            downloader.init();
```

Comment:
Another, possibly better, option would be to scope `downloader` to the inner `describe` block, and drop this `afterEach` since `beforeEach` creates a new one for each test.